processing file asmFiles/break.asm (1 of 1)
processing file asmFiles/HazardCF.asm (1 of 3)
processing file asmFiles/HazardData.asm (2 of 3)
processing file asmFiles/HazardStr.asm (3 of 3)
processing file asmFiles/assoc.asm (1 of 37)
processing file asmFiles/beq1.asm (2 of 37)
processing file asmFiles/beq2.asm (3 of 37)
processing file asmFiles/bge1.asm (4 of 37)
processing file asmFiles/bge2.asm (5 of 37)
processing file asmFiles/bgeu1.asm (6 of 37)
processing file asmFiles/bgeu2.asm (7 of 37)
processing file asmFiles/blt1.asm (8 of 37)
processing file asmFiles/blt2.asm (9 of 37)
processing file asmFiles/bltu1.asm (10 of 37)
processing file asmFiles/bltu2.asm (11 of 37)
processing file asmFiles/bne1.asm (12 of 37)
processing file asmFiles/bne2.asm (13 of 37)
processing file asmFiles/break.asm (14 of 37)
processing file asmFiles/count_days.asm (15 of 37)
processing file asmFiles/fib.asm (16 of 37)
--- memsim.hex	2025-10-29 20:24:42.741434500 -0400
+++ memcpu.hex	2025-10-29 20:24:42.736434308 -0400
@@ -37,5 +37,5 @@
 :040036000000452F52
 :0400370000006FF165
 :0403C0000000001623
-:040C4000000000367A
+:040C4000000000218F
 :00000001FF
processing file asmFiles/halt.asm (17 of 37)
processing file asmFiles/HazardCF.asm (18 of 37)
processing file asmFiles/HazardData.asm (19 of 37)
processing file asmFiles/HazardStr.asm (20 of 37)
processing file asmFiles/ibd.asm (21 of 37)
processing file asmFiles/idSC.asm (22 of 37)
processing file asmFiles/ihit_before_dhit.asm (23 of 37)
processing file asmFiles/ihit_dhit_same.asm (24 of 37)
processing file asmFiles/jal.asm (25 of 37)
processing file asmFiles/jalr2.asm (26 of 37)
processing file asmFiles/jalr.asm (27 of 37)
processing file asmFiles/mergesort.asm (28 of 37)
--- memsim.hex	2025-10-29 20:25:11.288531574 -0400
+++ memcpu.hex	2025-10-29 20:25:11.281531305 -0400
@@ -215,6 +215,6 @@
 :04017D00000000631B
 :04017E000000006419
 :04017F000000006418
-:040C40000000057437
+:040C40000000054566
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (29 of 37)
processing file asmFiles/mult_procedure.asm (30 of 37)
processing file asmFiles/search.asm (31 of 37)
processing file asmFiles/test.branchjump1.asm (32 of 37)
processing file asmFiles/test.branchjump2.asm (33 of 37)
processing file asmFiles/test.ldst1.asm (34 of 37)
processing file asmFiles/test.ldst2.asm (35 of 37)
processing file asmFiles/test.loadstore.asm (36 of 37)
processing file asmFiles/test.rtype.asm (37 of 37)
